src/huggingface_hub/_login.py

```diff
@@ -35,7 +35,7 @@
 logger = logging.get_logger(__name__)
 
 
-def login(token: Optional[str] = None, add_to_git_credential: bool = True) -> None:
+def login(token: Optional[str] = None, add_to_git_credential: bool = False) -> None:
```

Comment:
i'm ok with switching that back to `True`

Reply:
Lol, let's keep as it is know (`False`). The user will be warned anyway.